src/testing/core/__tests__/createTestSchema.test.tsx

```diff
@@ -1028,4 +1028,192 @@ describe("schema proxy", () => {
 
     unmount();
   });
+
+  it("createSchemaFetch respects min and max delay", async () => {
+    const Profiler = createDefaultProfiler<ViewerQueryData>();
+
+    const maxDelay = 2000;
+
+    using _fetch = createSchemaFetch(schema, {
+      delay: { min: 10, max: maxDelay },
+    });
+
+    const client = new ApolloClient({
+      cache: new InMemoryCache(),
+      uri,
+    });
+
+    const query: TypedDocumentNode<ViewerQueryData> = gql`
+      query {
+        viewer {
+          id
+          name
+          age
+          book {
+            id
+            title
+            publishedAt
+          }
+        }
+      }
+    `;
+
+    const Fallback = () => {
+      useTrackRenders();
+      return <div>Loading...</div>;
+    };
+
+    const App = () => {
+      return (
+        <React.Suspense fallback={<Fallback />}>
+          <Child />
+        </React.Suspense>
+      );
+    };
+
+    const Child = () => {
+      const result = useSuspenseQuery(query);
+
+      useTrackRenders();
+
+      Profiler.mergeSnapshot({
+        result,
+      } as Partial<{}>);
+
+      return <div>Hello</div>;
+    };
+
+    const { unmount, rerender } = renderWithClient(<App />, {
+      client,
+      wrapper: Profiler,
+    });
+
+    // initial suspended render
+    await Profiler.takeRender();
+
+    {
+      try {
+        const { snapshot: _snapshot } = await Profiler.takeRender();
```

Comment:
```suggestion
        await Profiler.takeRender();
```

No need to assign anything since we aren't checking against the snapshot 🙂

Reply:
Removed https://github.com/apollographql/apollo-client/pull/11774/commits/def4b0d4c93ad0ae4120b902ec39236fbc8c50ea